app/controllers/api/v2/job_templates_controller.rb

```diff
@@ -108,9 +108,7 @@ def destroy
       param :id, :identifier, :required => true
       param_group :job_template_clone, :as => :create
       def clone
-        @job_template = @job_template.clone
-        load_vars_from_template
-        @job_template.name = job_template_params[:name]
+        @job_template = @job_template.clone_job_template(job_template_params[:name])
```

Comment:
This all still feels a bit heavy handed, couldn't we just set locked to false before saving it and call it a day?

Reply:
@adamruzicka Not exactly, I tried this earlier to add it here only but it doesn't work as expected, and not all associations like taxonomies are copied, which we need handle explicitely in clone_job_template method